src/components/map/Map.tsx

```diff
@@ -40,9 +97,14 @@ const Map: React.FC<MapComponentProps> = ({
     return <div>Loading Google Maps...</div>;
```

Comment:
Please use the `<Loader/>` component that is already made instead of text for a loading indicator